.github/workflows/validate-nginx-config.yml

```
@@ -0,0 +1,63 @@
+name: Validate Nginx Configuration
+on:
+  workflow_call:
+    inputs:
+      port:
+        default: 8080
```

actionlint (1.7.4)

input "port" of workflow_call event has the default value "8080", but it is also required. if an input is marked as required, its default value will never be used

(events)